README.win32

```diff
@@ -161,12 +164,12 @@ contributed his changes to the OCaml project.
 
 ------------------------------------------------------------------------------
 
-           The native Win32 port built with Mingw
-           --------------------------------------
+           The native Win32 and Win64 ports built with Mingw
+           -------------------------------------------------
 
 REQUIREMENTS:
 
-This port runs under MS Windows Seven, Vista, XP, and 2000.
+Thoss ports run under MS Windows Seven, Vista, XP, and 2000.
```

Comment:
Oups. 

I'm not aware of github workflow. Do you want me to provide a new pull request?